### PR TITLE
New stable version of tracim: release_02.06.01

### PR DESCRIPTION
**Pull request by PhilippeAccorsi**
#2682
- [x]  changelog and backend/frontend version is update

## Checkpoints

The goal of this section is to help code reviewer to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

- [ ] Tested by quality team

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -158,16 +168,17 @@ Archive button not more used in Tracim. All archive buttons is hide (#2347)
...
+  - lite search similar to Tracim v1
   - smart search base on ElasticSearch (full text) ([Documentation here](https://github.com/tracim/tracim/blob/develop/backend/doc/setting.md#search-method-using-elastic_search-tracim-23))
 - Easy content move using Drag & Drop
 - Lots of UX improvement
 
 ### New Features (technical)
 
-- It's now possible to configure tracim using environment variables ([Documentation here](https://github.com/tracim/tracim/blob/develop/backend/doc/setting.md#tracim-22-fully-supported-var))
+- It's now possible to configure Tracim using environment variables ([Documentation here](https://github.com/tracim/tracim/blob/develop/backend/doc/setting.md#tracim-22-fully-supported-var))
+If you have used Tracim on Docker before this release, you need to delete this files `tracim_web.ini` `tracim_web.ini` `tracim_caldav.ini` available in this default folder `~/tracim/etc` before starting Tracim 2.3.0 (all files are created on startup)
```
Comment (Skylsmoi) [inline changelog.md]: This part should be in migration guide part

Comment (PhilippeAccorsi) [reply changelog.md]: Done

File: changelog.md [hunk 2]
```diff
@@ -158,19 +168,23 @@ Archive button not more used in Tracim. All archive buttons is hide (#2347)
...
+- It's now possible to configure Tracim using environment variables ([Documentation here](https://github.com/tracim/tracim/blob/develop/backend/doc/setting.md#tracim-22-fully-supported-var))
 - New licence: combination of MIT, LGPLv3 and AGPLv3
-- Since 2.3 version, following services are automatically launched when tracim start with docker:
+- Since 2.3 version, following services are automatically launched when Tracim start with docker:
   - Radicale (agenda)
   - WebDAV
 
+### Migration from <= 2.2.0 to 2.3.0
+
+- Docker: you need to delete this files `tracim_web.ini` `tracim_web.ini` `tracim_caldav.ini` available in this default folder `~/tracim/etc` before starting Tracim (all files are created on startup)
```
Comment (Skylsmoi) [inline changelog.md]:
```suggestion
- Only if you are using Docker: before starting Tracim, you need to delete these files `tracim_web.ini` ===> missing webdav file <=== `tracim_caldav.ini` available in the default folder `~/tracim/etc` (all files are created on startup)
```

Comment (PhilippeAccorsi) [reply changelog.md]: Thats a good suggestion.

Comment (PhilippeAccorsi) [reply changelog.md]: Done

**Pull request merged**